doc/whats_new/v0.22.rst

```diff
@@ -299,3 +299,15 @@ These changes mostly affect library developers.
 - |Enhancement| Binary only classifiers are now supported in estimator checks.
   Such classifiers need to have the `binary_only=True` estimator tag.
   :pr:`13875` by `Trevor Stephens`_.
+
+- Created two new multioutput estimator tests:
+  :func:`~utils.estimator_checks.check_classifier_multioutput` and
+  :func:`~utils.estimator_checks.check_regressor_multioutput`.
+  Added them to :func:`~utils.estimator_checks._yield_classifier_checks` and
```

Comment:
This is not public API. Basically they are included in check_estimator

Reply:
Got it. Shortened it, please review.